capabilityquerier/cmd/main.go

```diff
@@ -24,15 +25,16 @@ import (
 // (see endpointmanager/pkg/workers) as well as the arguments for the capabilityquerier.QuerierArgs
 // struct that is used when calling capabilityquerier.GetAndSendCapabilityStatement
 type queryArgs struct {
-	workers     *workers.Workers
-	ctx         context.Context
-	client      *http.Client
-	jobDuration time.Duration
-	mq          *lanternmq.MessageQueue
-	ch          *lanternmq.ChannelID
-	qName       string
-	userAgent   string
-	store       *postgresql.Store
+	workers        *workers.Workers
+	ctx            context.Context
+	client         *http.Client
+	jobDuration    time.Duration
+	mq             *lanternmq.MessageQueue
+	ch             *lanternmq.ChannelID
+	qName          string
+	userAgent      string
+	store          *postgresql.Store
+	exportFileWait int
```

Comment:
Can this just be grabbed from the environment variable rather than being sent as a message parameter each time?